workspace.json

```diff
@@ -6800,7 +6800,7 @@
         "seed": {
           "builder": "@nrwl/workspace:run-commands",
           "options": {
-            "command": "../../../node_modules/.bin/sequelize-cli db:seed",
+            "command": "../../../node_modules/.bin/sequelize-cli db:seed í ../../../node_modules/.bin/sequelize-cli db:seed --seed=20210514094740-development.js",
```

Comment:
Is this correct?

Reply:
Yes, seed is defined here to ease development it is currently not used in production.
Else the seeder will run the e2e seeds as well

Reply:
There is a `í` in the middle there